docs/minimal/mysql.md

```diff
@@ -59,11 +59,15 @@ To run a minimal downtime schema and data migration:
 
 ## UI
 
+{: .important }
+Spanner migration tool supports sharded MySQL migrations. Follow the instructions below to configure sharded migrations.
+
 Follow the steps below to configure a minimal downtime migration are MySQL:
 
 1. [Connect to source database](../ui/connect-source.md).
 2. [Connect to spanner instance](../ui/connect-spanner.md).
-3. [Convert MySQL schema to Spanner schema using the schema conversion workspace](../ui/schema-conv/schema-conv.md). Follow the documentation in the schema conversion workspace section of the documentation for different schema modifications that are supported by SMT. For guidance on specific data type conversion for Oracle, look at the [data conversion](../data-types/mysql.md) for MySQL documentation.
-4. [Configure datastream and dataflow details](../ui/prepare-migration/prepare.md).
-5. Wait for the migration to be orchestrated by SMT.
-6. Look at the list of [generated resources](../ui/prepare-migration/monitor.md/#generated-resources) for links to the Datastream stream and the Dataflow job crearted.
+3. [Convert MySQL schema to Spanner schema using the schema conversion workspace](../ui/schema-conv/schema-conv.md). 
+4. Follow the documentation in the schema conversion workspace section of the documentation for different schema modifications that are supported by SMT. For guidance on specific data type conversion for MySQL, look at the [data conversion](../data-types/mysql.md) for MySQL documentation. Note that a new column, `migration_shard_id` gets added to each table for a sharded migration. More on that [here](../ui/schema-conv/sharded-migration.md#addition-of-migration_shard_id).
```

Comment:
Maybe migration shard_shard_id fits better in the sharded migration section ?

Reply:
Done